website/docs/components/code-editor/partials/code/component-api.md

```diff
@@ -0,0 +1,90 @@
+  </C.Property>
+  <C.Property @name="ariaLabelledBy" @type="string">
+    Accepts a localized string. The `ariaLabelledBy` value is applied to the code editor input element.
+  </C.Property>
+  <C.Property @name="hasCopyButton" @type="boolean" @default="false">
+    Used to control whether a copy button for copying the code/text content will be displayed.
+  </C.Property>
+  <C.Property @name="hasFullScreenButton" @type="boolean" @default="false">
+    Used to control whether a toggle button for toggling full-screen mode will be displayed.
+  </C.Property>
+  <C.Property @name="isStandalone" @type="boolean" @default="true">
+    Applies rounded borders to the component. When used within another component or when the context requires it, you can turn it off.
+  </C.Property>
+  <C.Property @name="language" @type="string" @values={{array "go" "hcl" "json" "ruby" "sentinel" "shell" "sql" "yaml"}}>
+    The coding language to use for syntax highlighting. If you need additional languages <LinkTo class="doc-link-generic" @route="show" @model="about/support">contact the Design Systems Team</LinkTo>.
+  </C.Property>
+  <C.Property @name="value" @type="string">
+    The text/code content for the Code Editor.
+  </C.Property>
+  <C.Property @name="onBlur" @type="function">
+    Callback function invoked when focus is removed from the editor.
+  </C.Property>
+  <C.Property @name="onInput" @type="function">
+    Callback function invoked when the editor receives an input event.
+  </C.Property>
+  <C.Property @name="onSetup" @type="function">
+    Callback function invoked when the editor completes setup.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Contextual components
+
+#### [CE].Title
+
+The `CodeEditor::Title` component, yielded as contextual component.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Accepts complex content, such as logic/conditionals, HTML elements, other Ember components, etc. Styling is applied for simple HTML elements, such as `strong`, `em`, `a`, `code/pre`. Consumers will need to style other HTML tags if used as children.
+  </C.Property>
+    <C.Property @name="tag" @type="enum" @values={{array "p" "h1" "h2" "h3" "h4" "h5" "h6"}} @default="h2">
+    The HTML tag that wraps the content of the "title" block.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+#### [CE].Description
+
+The `CodeEditor::Description` component, yielded as contextual component.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Accepts complex content, such as logic/conditionals, HTML elements, other Ember components, etc. Styling is applied for simple HTML elements, such as `strong`, `em`, `a`, `code/pre`. Consumers will need to style other HTML tags if used as children
```

Comment:
I still think we should rethink these sections discussing styling but since similar language is included in the CodeBlock docs we could look at it at the same time.

(We do not include any styling for strong, em, anchor tags, etc. Any styles applied to those would come from either browser defaults or default styles consumers may have set.)

Reply:
Also by the way, as the description is marked up a a `p` tag there's a limit to the types of HTML content it can contain so it actually shouldn't contain overly complex content.